### PR TITLE
Mohamed chat messages front end components

### LINKED ISSUE #55: Frontend ChatDisplay Components
Part of #5, implements Fixtures, cards and stack for messages

### PR DESCRIPTION
**Pull request by Mohamed-Elfouly**
Implements the first of the front-end components for the chat message feature, Fixtures, the cards for the messages and the stack for them. There is also a storybook.

image

closes #55

### CODE REVIEW

**Review by cjyu81 — DISMISSED** [review]
Tests pass, looks good. Approving.

**Review by pconrad — APPROVED** [review]
LGTM as a first step.

I suggest that in a later PR, if you haven't already, that you take these steps:
1. Create a MessageDTO (data transfer object) that includes both the message, and extra fields from user that you want to send (e.g. given name, family name, email). I can go over both approaches with you in class or over chat.
2. Add either the users email or given name/family name to the message along with the user id.  
2.

**Comment by Mohamed-Elfouly** [comment]
> LGTM as a first step.
> 
> I suggest that in a later PR, if you haven't already, that you take these steps:
> 
> 1. Create a MessageDTO (data transfer object) that includes both the message, and extra fields from user that you want to send (e.g. given name, family name, email). I can go over both approaches with you in class or over chat.
> 2. Add either the users email or given name/family name to the message along with the user id.

I have not done that. I would greatly appreciate the help in class

**Comment by Mohamed-Elfouly** [comment]
Although I think these files should change, I want to consult the professor or TAs

**Comment by Mohamed-Elfouly** [comment]
Also the Resolve conflicts button is disabled for some reason

**Review by andrewthaii — DISMISSED** [review]
LGTM

File: frontend/src/stories/components/ChatMessage/ChatDisplay.stories.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { rest } from "msw";
+
+export default {
+  title: 'components/ChatDisplay',
+  component: ChatDisplay,
+};
+
+const Template = (args) => <ChatDisplay {...args} />;
+
+//const Template = () => <HelpRequestCreatePage storybook={true} />;
```
Comment (pconrad) [inline frontend/src/stories/components/ChatMessage/ChatDisplay.stories.js:15]:
```suggestion
```

**Review by pconrad — CHANGES_REQUESTED** [review]
One small change

**Review by pconrad — APPROVED** [review]
LGTM

**Review by abcdefucsb — APPROVED** [review]
This code has been reviewed by others. LGTM!

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**